locobot/agent/locobot_mover_utils.py

```diff
@@ -63,11 +63,44 @@ def transform_pose(XYZ, current_pose):
     return XYZ
 
 
+def get_move_target_for_point(base_pos, target, yaw, eps=1):
+    """
```

Comment:
could we separate out the logic for where to move for the point (from the coordinate transforms) and put it in tasks.py?

Reply:
The util still lives here (will move this to a better named utils file in the medium term) but gets called in tasks.py. Does that work?

Reply:
yeah, as long as the tasks steps are controlled by the agent loop I am fine with merging.  I would rather the logic for choosing where to move is in an interpreter... why a util file?  will this logic be used somewhere else?